app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+require 'rack-flash'
+class TweetsController < ApplicationController
+
+use Rack::Flash
+
+  get '/tweets' do
+  	if logged_in?
+      @user = User.find_by_id(session[:user_id])
+      @tweets = Tweet.all
+  		erb :'/tweets/tweets'
+  	else
+  		redirect "/login"
+  	end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'/tweets/new'
+    else
+      redirect "/login"
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      flash[:message] = "A tweet cannot be empty. Please fill in some content."
+      redirect '/tweets/new'
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.new(content: params[:content], user_id: session[:user_id])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.